src/libraries/Microsoft.Extensions.Http/src/Logging/LogHelper.cs

```diff
@@ -32,6 +31,8 @@ private static class EventIds
             public static readonly EventId RequestPipelineResponseHeader = new EventId(103, "RequestPipelineResponseHeader");
         }
 
+        public static readonly Func<string, bool> ShouldRedactHeaderValue = (header) => true;
```

Comment:
NIT: Why is it public write, when nothing is changing it ATM?

Reply:
> public write

Can you elaborate pls? I'm not sure I follow. This is a readonly field... 🤔

It is public because there were other methods in this internal class that were public, like `public static void LogRequestStart`, but I can make it internal if you feel strongly about it

Reply:
Oops... my bad. I missed the `readonly`, sorry